src/BlockListener.php

```diff
@@ -19,4 +19,31 @@ public function type(): int
     {
         return self::TYPE_BLOCK;
     }
+
+    /**
+     * @param Line - An array with Line objects
+     * @return Line
+     * @since 1.3.2
+     */
+    public function getFirstLine($pick)
```

Comment:
The docblock suggests it is a Line, but you're passing a Pick right? Nice to also add a type hint to make sure it is correct.

And maybe good to make this protected.